packages/@webex/plugin-meetings/src/locus-info/selfUtils.ts

```diff
@@ -428,11 +428,6 @@ SelfUtils.mutedByOthersChanged = (oldSelf, changedSelf) => {
     return false;
   }
 
-  // there is no need to trigger user update if no one muted user
-  if (changedSelf.selfIdentity === changedSelf.modifiedBy) {
-    return false;
-  }
-
```

Comment:
reverted because this was causing the BEMS bug

Reply:
note to self, ask about this if not resolved lower. edit: resolved in next file